extensions/amp-beopinion/validator-amp-beopinion.protoascii

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-beopinion
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-beopinion"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+    requires_usage: GRANDFATHERED
+    deprecated_allow_duplicates: true
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-beopinion>
+  html_format: AMP
+  tag_name: "AMP-BEOPINION"
+  requires_extension: "amp-beopinion"
+  attrs: {
+    name: "data-account"
```

Comment:
If `data-account` is always of a certain format, then that format could be enforced with a `value_regex_casei` set. E.g. if it was always a set of 5 numeric values then it would be `value_regex_casei: "[0-9]{5}"`. This is useful to help AMP content creators to know they entered the correct value.